bridge/bridge.go

```diff
@@ -97,8 +100,7 @@ func (b *Bridge) Sync(quiet bool) {
 
 	log.Printf("Syncing services on %d containers", len(containers))
 
-	// NOTE: This assumes reregistering will do the right thing, i.e. nothing.
-	// NOTE: This will NOT remove services.
+	// NOTE: This assumes reregistering will do the right thing, i.e. nothing..
```

Comment:
This re-registration will cause problem in some cases. For example we use this to auto-register our services in docker, and will also register HTTP check (ping), this re-registration will cause the service to be marked as down for a short time, and cause our LB to bouncing the service back and force...Why not making a flag to disable/enable it?